Issue463 abstract test case tests

EttoreZ opened the pull request:
This pull request is to address issue #463.

dhblum commented on testing/test_bestest_air.py:
```diff
@@ -60,6 +60,8 @@ def setUp(self):
         self.url = 'http://127.0.0.1:5000'
         self.step_ref = 3600
         self.test_time_period = 'peak_heat_day'
+        self.input = {'fcu_oveTSup_activate': 0, 'fcu_oveTSup_u': 290}
```
@EttoreZ I suggest calling "input_false_overwrite" or add a comment that the value of activate is meant to be 0 for the test_advance_false_overwrite API test.

dhblum reviewed (changes requested):
@EttoreZ Thanks for this PR.  Please address the one inline comment and then I think this will be ready to go.

EttoreZ commented:
Thank you @dhblum for the careful review. I added the comment `#<u_variable>_activate is meant to be 0 for the test_advance_false_overwrite API test` above the self.input definition and updated the pull request.

dhblum reviewed (approved):
Thanks @EttoreZ for the udpates.
